tft.py

```diff
@@ -0,0 +1,213 @@
+from cdragontoolbox.binfile import BinFile, BinHashBase, BinHashValue
+from json import JSONEncoder
+import json
+import mmap
+import copy
+import re
+import os
+
+
+class NaiveJsonEncoder(JSONEncoder):
+    def default(self, other):
+        if isinstance(other, BinHashBase):
+            if other.s is None:
+                return other.hex
+            return other.s
+        return other.__dict__
+
+
+class Transformer:
+
+    output_prefix = r"D:\pbe\tft_temp"
+    prefix = r"D:\pbe"
+    map22_file = os.path.join(prefix, r"data\maps\shipping\map22\map22.bin")
+    font_folder = os.path.join(prefix, r"data\menu")
+    character_folder = os.path.join(prefix, r"data\characters")
+    exported_file = os.path.join(prefix, r"files.exported.txt")
```

Comment:
- Don't put hardcoded paths
- Use forward slashes as path delimiters (they are supported on both Windows on Linux so it's ok).

Reply:
if you dont hardcode paths, how are you supposed to refer to those files?
(atleast the bottom 4, since they are static in cdragon)